## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 458917`
Commit: `ad90d6231b685f8737a490abb4211cb133839437`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.slack</li>
    <li>ntfy</li>
    <li>ntfy.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.slack-bolt</li>
    <li>python312Packages.slack-bolt.dist</li>
    <li>python312Packages.slack-sdk</li>
    <li>python312Packages.slack-sdk.dist</li>
    <li>python313Packages.slack-bolt</li>
    <li>python313Packages.slack-bolt.dist</li>
    <li>python313Packages.slack-sdk</li>
    <li>python313Packages.slack-sdk.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.slack</li>
    <li>ntfy</li>
    <li>ntfy.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.slack-bolt</li>
    <li>python312Packages.slack-bolt.dist</li>
    <li>python312Packages.slack-sdk</li>
    <li>python312Packages.slack-sdk.dist</li>
    <li>python313Packages.slack-bolt</li>
    <li>python313Packages.slack-bolt.dist</li>
    <li>python313Packages.slack-sdk</li>
    <li>python313Packages.slack-sdk.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>ntfy</li>
    <li>ntfy.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python313Packages.slack-bolt</li>
    <li>python313Packages.slack-bolt.dist</li>
    <li>python313Packages.slack-sdk</li>
    <li>python313Packages.slack-sdk.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.slack-bolt</li>
    <li>python312Packages.slack-bolt.dist</li>
    <li>python312Packages.slack-sdk</li>
    <li>python312Packages.slack-sdk.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>ntfy</li>
    <li>ntfy.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.slack-bolt</li>
    <li>python312Packages.slack-bolt.dist</li>
    <li>python312Packages.slack-sdk</li>
    <li>python312Packages.slack-sdk.dist</li>
    <li>python313Packages.slack-bolt</li>
    <li>python313Packages.slack-bolt.dist</li>
    <li>python313Packages.slack-sdk</li>
    <li>python313Packages.slack-sdk.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python313Packages.slack-sdk</summary>
<pre>
tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_coverage
tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_legacy_coverage
  /nix/var/nix/builds/nix-15160-2156325322/source/slack_sdk/web/internal_utils.py:298: UserWarning: The top-level `text` argument is missing in the request payload for a chat.postEphemeral call - It&#x27;s a best practice to always provide a `text` argument when posting a message. The `text` argument is used in places where content cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
    warnings.warn(missing_text_message, UserWarning)

tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_coverage
tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_legacy_coverage
  /nix/var/nix/builds/nix-15160-2156325322/source/slack_sdk/web/internal_utils.py:298: UserWarning: The top-level `text` argument is missing in the request payload for a chat.postMessage call - It&#x27;s a best practice to always provide a `text` argument when posting a message. The `text` argument is used in places where content cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
    warnings.warn(missing_text_message, UserWarning)

tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_coverage
tests/slack_sdk_async/web/test_web_client_coverage.py::TestWebClientCoverage::test_legacy_coverage
  /nix/var/nix/builds/nix-15160-2156325322/source/slack_sdk/web/internal_utils.py:298: UserWarning: The top-level `text` argument is missing in the request payload for a chat.update call - It&#x27;s a best practice to always provide a `text` argument when posting a message. The `text` argument is used in places where content cannot be rendered such as: system push notifications, assistive technology such as screen readers, etc.
    warnings.warn(missing_text_message, UserWarning)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/web/test_web_client.py::TestWebClient::test_issue_690_oauth_access - ConnectionResetError: [Errno 54] Connection reset by peer
= 1 failed, 869 passed, 5 skipped, 26 deselected, 123 warnings in 244.35s (0:04:04) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/slack-sdk/default.nix
+++ b/pkgs/development/python-modules/slack-sdk/default.nix
@@ -13,7 +13,6 @@
   sqlalchemy,
   websocket-client,
   websockets,
-  writableTmpDirAsHomeHook,
 }:
 
 buildPythonPackage rec {
@@ -54,6 +53,14 @@ buildPythonPackage rec {
     "test_start_raises_an_error_if_rtm_ws_url_is_not_returned"
     # Requires network access: [Errno 111] Connection refused
     "test_send_message_while_disconnection"
+    # [Errno 61] Connection refused
+    "TestWebClient_HttpRetry"
+    # ConnectionResetError: [Errno 54] Connection reset by peer
+    "test_issue_690_oauth_access"
+    "test_issue_690_oauth_v2_access"
+    "test_error_response"
+    # Assertion Error: unexpectedly None (while accessing instance of slack store)
+    "test_issue_1441_mixing_user_and_bot_installations"
   ];
 
   disabledTestPaths = [

--- a/pkgs/development/python-modules/slack-sdk/default.nix
+++ b/pkgs/development/python-modules/slack-sdk/default.nix
@@ -51,15 +51,11 @@ buildPythonPackage rec {
   disabledTests = [
     # Requires internet access (to slack API)
     "test_start_raises_an_error_if_rtm_ws_url_is_not_returned"
-    # Requires network access: [Errno 111] Connection refused
     "test_send_message_while_disconnection"
-    # [Errno 61] Connection refused
     "TestWebClient_HttpRetry"
-    # ConnectionResetError: [Errno 54] Connection reset by peer
     "test_issue_690_oauth_access"
     "test_issue_690_oauth_v2_access"
     "test_error_response"
-    # Assertion Error: unexpectedly None (while accessing instance of slack store)
     "test_issue_1441_mixing_user_and_bot_installations"
   ];
 

